README.md

```diff
@@ -89,13 +89,20 @@ service_wrapper = service_manager.create_service(
     # Factory function returning instance of yapapi_service_manager.ServiceWrapper
     # Sample usage --> Erigon example
     service_wrapper_factory=yapapi_service_manager.ServiceWrapper,
+    
+    # Optional dictionary of parameters passed directly to Golem.run_service
+    # (e.g. network or instance_params). IMPORTANT NOTE: some params might influence the number
+    # of instances created. This must be avoided, exactly one instance should be always created.
+    run_service_params=None,
```

Comment:
considering your comment above, wouldn't it make sense to always pass `num_instances=1` ?

Reply:
Good idea! Done in e11266d.